ingestion/src/metadata/utils/test_connection_clients.py

```diff
@@ -0,0 +1,24 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from dataclasses import dataclass
+
+
+@dataclass
+class GlueClient:
+    def __init__(self, client) -> None:
+        self.client = client
```

Comment:
I like the idea of moving this to its own module. A couple of comments though:

The `dataclass` would be better defined as:

```
@dataclass
class GlueClient:
    client: AWSClient
```

Also, the module name makes it look like it might be a test module. I'd just name it `connection_clients`.
Thanks!

Reply:
@pmbrull, will rename it  to `connection_clients`.


```
@dataclass
class GlueClient:
    client: AWSClient
```
This would actually throw a circular dependency error, I logged the type of glue client it's `<class 'botocore.client.Glue'>`  seems like this is getting generated on the fly, because the import for this class is not available

same for the dynamodb with `<class 'boto3.resources.factory.dynamodb.ServiceResource'>`

Reply:
also if I import through the boto client then again there will be possibility of import error right 🤔

Reply:
yeah np, let's keep it as is to minimize current issues. We can give it another go if necessary in further releases. Thanks 🙏